example/default/example.dart

```diff
@@ -50,15 +50,16 @@ void runFailureCase() {
 
   HttpRequest.getString('http://httpstat.us/500').then((String result) {
     span.log('data_received', payload: result);
-  }).catchError((error) {
+  }).catchError((dynamic error) {
```

Comment:
Dart recommends not specifying types in these anonymous functions. Is there a reason we are declaring this as `dynamic`? If so, would it be better to inline a type cast to be more explicit?

Reply:
I was getting an analyzer warning to 'Specify type annotations'.